### ISSUE TITLE
Slider for EXP/Sepith adjustment instead of fixed values

### ISSUE THREAD

**Issue by ghost** [nnguyen259/ColdSteelRandomizer#24]
Currently, you can only increase the Exp and sepith gain by a hardcoded value. Wouldn't a slider within a range (something like 5x to 0.2x for people who want a challenge) be better?

**Comment by nnguyen259** [comment]
Yes I agree. It's something that has been in the back burner, I'll look into implementing it.
